include/engine/api/table_parameters.hpp

```diff
@@ -135,6 +140,9 @@ struct TableParameters : public BaseParameters
         if (fallback_speed < 0)
             return false;
 
+        if (scale_factor < 1)
```

Comment:
Docs say > 0, but code enforces > 1.

I think any value should be allowed - `scale_factor=-0.5` is weird, but possible to imagine.

Reply:
So we shouldn't really enforce anything here?

Reply:
Yeah - I can’t think of a reason to restrict this, who knows how people might want to use it.

Reply:
@danpat I'm actually going to clamp to >0.  See https://github.com/Project-OSRM/osrm-backend/pull/5298#discussion_r240010511 for overflow checks and https://github.com/Project-OSRM/osrm-backend/pull/5298#issuecomment-445477266 for why doubling the number of overflow checks would trigger computation growth.